syft/frameworks/torch/tensors/interpreters/Polynomial.py

```diff
@@ -0,0 +1,54 @@
+from syft.frameworks.torch.tensors.interpreters.abstract import AbstractTensor
+
+
+class PolynomialTensor(AbstractTensor):
+
+    """Tensor type which provides polynomial approximation functionalities using Taylor Series expansion
+       since computing exact functions could impose a overhead on computation
+    """
+
+    def sigmoid(self, x: float) -> float:
+
+        """Parameters:
+            
+           x: A float value
+           
+           return: 
+               
+           approximation of the sigmoid function as a float"""
+
+        return (1 / 2) + (x / 4) - (x ** 3 / 48) + (x ** 5 / 480)
+
+    def exp(self, x: float) -> float:
```

Comment:
Same as above, this should operate on self. See add.py for examples for methods like __add__ (or mul as soon as I finish debugging it)